docs/v13/documentation/add-change-nunjucks-components.md

```diff
@@ -103,3 +103,39 @@ It is important that you do not change or delete the closing brackets:
  ]
 }) }}
 ```
+
+Your code should look like this:
+```
+{{ govukRadios({
+  name: "where-do-you-live",
+  fieldset: {
+    legend: {
+      text: "Where do you live?",
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  items: [
+    {
+      value: "england",
+      text: "England"
+    },
+    {
+      value: "scotland",
+      text: "Scotland"
+    },
+    {
+      value: "wales",
+      text: "Wales"
+    },
+    {
+      value: "northern-ireland",
```

Comment:
does it make sense to delete as per the instructions above? Or are we treating this as a separate instruction?